File: Java-Core/src/main/java/presto/type/MissingType.java
package presto.type;

import presto.runtime.Context;

public class MissingType extends NativeType {

	static MissingType instance = new MissingType();
	
	public static MissingType instance() {
		return instance;
	}
	
	private MissingType() {
		super("*");
	}
	
	@Override
	public Class<?> toJavaClass() {
		return Object.class;
	}

	@Override
	public boolean isAssignableTo(Context context, IType other) {
		return true;
	}
	
}

File: Java-CodeServer/src/main/resources/js/prompto/type/MissingType.js
var NativeType = require("./NativeType").NativeType;

function MissingType() {
	NativeType.call(this, "*");
	return this;
}

MissingType.prototype = Object.create(NativeType.prototype);
MissingType.prototype.constructor = MissingType;

MissingType.instance = new MissingType();
	
/*
	
	@Override
	public Class<?> toJavaClass() {
		return Object.class;
	}
*/

MissingType.prototype.isAssignableTo = function(context, other) {
	return true;
};

exports.MissingType = MissingType;